src/program.ts

```diff
@@ -210,11 +210,29 @@ function operatorKindFromDecorator(decoratorKind: DecoratorKind, arg: string): O
       break;
     }
     case CharCode.PLUS: {
-      if (arg.length == 1) return OperatorKind.ADD;
+      switch (arg) {
+        case "+":  return OperatorKind.ADD;
```

Comment:
Wouldn't this also trigger on `@operator.postfix("+")`? Might make sense to switch over `decoratorKind` early, and then check the strings per kind.

Reply:
Good catch!

Reply:
Fixed!

Reply:
About using switching OperatorKind at top level and after string matching, hmm... It will be little bit faster, yah.